services/services.c

```diff
@@ -25,31 +25,35 @@
 // note: use xstr() to convert the SERVICE_NAME_ #define from services.h to a usable string
 // (e.g. xstr(SERVICE_NAME_HEARTBEAT))
 const service_desc_t service_descriptors[] = {
-    {
-        .name = xstr(SERVICE_NAME_USB), 
-        .service_func = usb_service,
-        .startup = true
-    },
-    {
-        .name = xstr(SERVICE_NAME_CLI), 
-        .service_func = cli_service,
-        .startup = true
-    }, 
-    {
-        .name = xstr(SERVICE_NAME_STORMAN), 
-        .service_func = storman_service,
-        .startup = true
-    }, 
-    {
-        .name = xstr(SERVICE_NAME_WATCHDOG), 
-        .service_func = watchdog_service,
-        .startup = true
-    }, 
-    {
-        .name = xstr(SERVICE_NAME_HEARTBEAT), 
-        .service_func = heartbeat_service,
-        .startup = false
-    }
+        {
```

Comment:
Looks like tabs got jacked up here can you de-indent 4 spaces? #nitpick

Reply:
I mean, I _can_, but aren't continuations a double-indent everywhere else in the project?

Reply:
I don't think so? Not a big deal for now